assets/js/blocks/active-filters/block.js

```diff
@@ -101,16 +114,7 @@ const ActiveFiltersBlock = ( {
 						</Fragment>
 					) }
 				</ul>
-				<button
-					className="wc-block-active-filters__clear-all"
-					onClick={ () => {
-						setMinPrice( null );
-						setMaxPrice( null );
-						setProductAttributes( [] );
-					} }
-				>
-					{ __( 'Clear All', 'woo-gutenberg-products-block' ) }
-				</button>
+				{ hasFilters() ? clearAllButton : noFiltersLabel }
```

Comment:
![imatge](https://user-images.githubusercontent.com/3616980/72791279-524f0680-3c37-11ea-9448-fa61b9491744.png)

We should make sure the `noFiltersLabel` is not shown in the editor.

```suggestion
				{ hasFilters() || isEditor ? clearAllButton : noFiltersLabel }
```